code/modules/organs/internal/lungs/filter.dm

```diff
@@ -19,7 +21,7 @@
 		),
 		"plasma" = list(
 			new /datum/lung_gas/metabolizable("toxins", min_pp=16, max_pp=280),
-			new /datum/lung_gas/waste("oxygen",         max_pp=10),
+			new /datum/lung_gas/waste("carbon_dioxide",         max_pp=10),
```

Comment:
/datum/organ/internal/lungs/plasmaman
	name = "\improper Plasmaman lungs"
	removed_type = /obj/item/organ/lungs/plasmaman

	gasses = list(
		new /datum/lung_gas/metabolizable("toxins", min_pp=16, max_pp=140),
		new /datum/lung_gas/waste("carbon_dioxide",         max_pp=10),
		new /datum/lung_gas/sleep_agent("/datum/gas/sleeping_agent", trace_gas=1, min_giggle_pp=0.15, min_para_pp=1, min_sleep_pp=5),
	)

https://github.com/d3athrow/vgstation13/commit/2651e8a260cb3c4d4dd236ca0fbd85176815926d

It's an oversight that these make you breath out oxygen rather than CO2. I'll move it to another PR, but just want to put this out there